submissions/chabuChabu/javaScript/cost.js

```diff
@@ -0,0 +1,51 @@
+let quantity = document.getElementById("quantity")
+    productPrice = document.querySelector("#productPrice"),
+    shippingCost = document.getElementById("shippingCost"),
+    total = document.getElementById("productTotal"),
+    buyBtn = document.getElementById("submitButton");
+/*Calculates the cost of 1 product and populates fields on page loading*/
+total.innerHTML = parseInt(productPrice.innerHTML)+ parseInt(shippingCost.innerHTML);
+
+quantity.addEventListener("input",grandPrice);
+
+function grandPrice(){
```

Comment:
This function is doing so many things. Move line 13 - 26 into another function

Reply:
Comment addressed and second function implemented